Formula/h/hdf5.rb

```diff
@@ -113,5 +123,8 @@ def install
     EOS
     system "#{bin}/h5fc", "test.f90"
     assert_equal version.to_s, shell_output("./a.out").chomp
+
+    # Make sure that it was built with SZIP/libaec
+    system("#{bin}/h5cc -showconfig | grep -q -e 'I/O filters' -e 'DECODE'")
```

Comment:
```suggestion
    config = shell_output("#{bin}/h5cc -showconfig")
    assert_match  "I/O filters", config
    assert_match  "DECODE", config
```

Reply:
Regarding this: DECODE needs to be in the same line as I/O filters, therefore my initial approach of double grep. It feels like this would achieve sth different.
I'm happy to incorporate another suggestion though!

Reply:
how about using the regex option of `assert_match` then?

Reply:
Looks good!